EFCore.CheckConstraints/CheckConstraintsExtensions.cs

```diff
@@ -52,25 +93,115 @@ public static class CheckConstraintsExtensions
             return optionsBuilder;
         }
 
+        /// <summary>
+        ///     Adds or updates a <see cref="CheckConstraintsOptionsExtension"/>
+        ///     object with all kinds of check constraints enabled
+        ///     to the current <see cref="DbContextOptionsBuilder"/> object.
+        /// </summary>
+        /// <param name="optionsBuilder">
+        ///     <see cref="DbContextOptionsBuilder"/> object having <see cref="CheckConstraintsOptionsExtension"/>
+        ///     added or updated with all kinds of check constraints enabled.
+        /// </param>
+        /// <param name="validationCheckConstraintsOptionsAction">
+        ///     Configures validation check constraint creation.
+        /// </param>
+        /// <returns>
+        ///     The current <see cref="DbContextOptionsBuilder"/> object.
+        /// </returns>
         public static DbContextOptionsBuilder UseAllCheckConstraints(
-            [NotNull] this DbContextOptionsBuilder optionsBuilder)
+            [NotNull] this DbContextOptionsBuilder optionsBuilder,
+            [CanBeNull] Action<ValidationCheckConstraintOptionsBuilder> validationCheckConstraintsOptionsAction = null)
             => optionsBuilder
                 .UseEnumCheckConstraints()
                 .UseDiscriminatorCheckConstraints()
-                .UseValidationCheckConstraints();
-
+                .UseValidationCheckConstraints(validationCheckConstraintsOptionsAction);
+
```

Comment:
Nit: please remove extra empty lines, leaving only one

Reply:
I'll gladly do ...

I did this because my intention was to visually separate the generic overloads group from the implementation group.

Just to make sure: I used **3** blank lines to separate them. Would **2** lines be alright for you? Or do you want me to wipe the separation completely and use just **1** blank line?

Reply:
I'd rather we followed the existing styling (which also corresponds to EF Core itself) of just 1 blank line.

Reply:
Done.